src/chartiq_injections/plotterDrawText.js

```diff
@@ -0,0 +1,21 @@
+export const plotterDrawText = () => {
```

Comment:
@shayanbinary Would you please add some comments why the `drawText` has been completely overwritten instead of using injection for further references?